### DISCUSSION TITLE
Possibility of read-only records by default ?

### DISCUSSION THREAD

**Discussion by Madydri** [keepassxreboot/keepassxc#7372] [Q&A]
Hello,
I've had some cases where data in a record was changed by error (ctrl-X etc...)
Of course there are backups and history-functionality.
But is it possible to set the things so up that a record is showed by default in read-only mode,
and that an extra action is necessary to make it writable ?
Thanks

**Comment by droidmonkey** [comment]
This is not possible, but you can use the preview panel at the bottom of the window to view most entry data without opening it in edit mode.
